sheet.go

```diff
@@ -177,37 +177,38 @@ func (f *File) workSheetWriter() {
 
 // trimRow provides a function to trim empty rows.
 func trimRow(sheetData *xlsxSheetData) []xlsxRow {
-	var (
-		row  xlsxRow
-		rows []xlsxRow
-	)
-	for k, v := range sheetData.Row {
+	var row xlsxRow
```

Comment:
Declare variable in the same way we be better (rename nextIndex to i):

```go
var (
    row xlsxRow
    i   int
)
```

Reply:
Done.

Reply:
Thanks for your great work. Could you run a new pprof figure to show how much memory was reduced in the optimization?